template/v0/Dockerfile

```diff
@@ -62,6 +62,11 @@ RUN micromamba install -y --name base --file /tmp/$ENV_IN_FILENAME && \
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo ln -s $(which python3) /usr/bin/python
 
+# Enforce dependencies are all installed from conda-forge
+RUN conda config --remove channels defaults
+RUN micromamba config append channels conda-forge --env
+RUN micromamba config set channel_priority strict
```

Comment:
Each `RUN` command in docker file will generate a new layer in docker image. This might be responsible for the docker image size increase